src/assets/translations/szit.json

```diff
@@ -6406,12 +6406,12 @@
               "leading": true,
               "content": [
                 {
-                  "text": "Nem jut be mindenki a mennyek országába, aki mondja nekem: – Uram, Uram! Csak az, aki teljesíti mennyei Atyám akaratát. Sokan mondják majd nekem azon a napon:",
```

Comment:
In case PR is accepted:
https://github.com/anwolosz/synopsis/wiki/Differences-from-the-sources
has to be extended with.